src/runner.rs

```diff
@@ -139,9 +148,65 @@ fn parse_test(test_path: &Path, config: &TestConfig) -> InnerTestResult<Test> {
         expected_stdout,
         expected_stderr,
         expected_exit_status,
+        rest,
     })
 }
 
+fn overwrite_test(test_path: &PathBuf, config: &TestConfig, output: &Output, test: &Test) -> std::io::Result<()> {
+    // Maybe copy the file so we don't remove it if we fail here?
+    let mut file = File::create(test_path)?;
+
+    file.write_all(test.rest.trim_end().as_bytes())?;
+    writeln!(file, "")?;
+    writeln!(file, "")?;
+
+    if !test.command_line_args.is_empty() {
+        writeln!(file, "{}{}", config.test_args_prefix, test.command_line_args)?;
+        writeln!(file, "")?;
+    }
+
+    if Some(0) != output.status.code() {
+        writeln!(
+            file,
+            "{} {}",
+            config.test_exit_status_prefix,
+            output.status.code().unwrap_or(0)
+        )?;
+        writeln!(file, "")?;
```

Comment:
It shouldn't be necessary to add the extra newline after the args or exit status. It is only needed to separate stdout and stderr if stdout spans multiple lines.

Reply:
I thought it read better. But sure, it's gone.